spec/spec_helper.rb

```diff
@@ -2,3 +2,6 @@
 require_relative '../classes/music_album'
 require_relative '../classes/book'
 require_relative '../classes/label'
+require_relative '../classes/game'
+require_relative '../classes/author'
+require_relative '../classes/item'
```

Comment:
What do you think about using the `Game` instance to be checked on `author_spec` line `16` because the class `Game` is also a child of the class `Item`?

Reply:
@aimalamiri . Kindly check the updated changes in the PR. thanks for your feedback.